lib/spandex_tesla_distributed.ex

```diff
@@ -48,8 +48,14 @@ defmodule SpandexTeslaDistributed do
         try do
           case Tesla.run(Tesla.put_headers(env, headers), next) do
             {:ok, env} ->
-              tracer.update_span(http: [status_code: env.status])
-              {:ok, env}
+              case env.status do
+                status when status not in 200..299 ->
```

Comment:
Do we want to report 300..399 treated as an error? a 302 redirect doesn't seem like an error in this context? 
A similar argument for 4xx class status codes, but I see there's no warn level to use.

Reply:
I forgot to comment on this, because I agree.  

Anything using our web module redirects port 80 to port 443 with a 301 status code.  

I am not sure if we want 4xx's to be an error or an okay, neither feel right, but as you said - there are no warn levels..  So perhaps something like 404 should be `ok` but something like a 400 or 401 should be an error..

Reply:
Maybe we just want to do `400..599` as an error?
200 - 399 are not necessarily issues.
Traces only seem to have Ok / Error as status options so 4xx should probably just be considered an Error in that case.